cmake/BuildLuaCAPITests.cmake

```diff
@@ -0,0 +1,56 @@
+set(LUA_C_API_TESTS_TARGET lua-c-api-tests)
+# Beware: OSS Fuzz build script searches tests in a directory
+# `${PROJECT_BINARY_DIR}/test/fuzz`.
+set(LUA_C_API_TESTS_PREFIX ${PROJECT_BINARY_DIR}/test/fuzz/${LUA_C_API_TESTS_TARGET})
+
+string(JOIN ";" LUA_C_API_TESTS
+  ffi_cdef_proto_test
+  lua_dump_test
+  luaL_gsub_test
+  luaL_loadbuffer_proto_test
```

Comment:
Removed:

```diff
@@ -3,11 +3,13 @@ set(LUA_C_API_TESTS_TARGET lua-c-api-tests)
 # `${PROJECT_BINARY_DIR}/test/fuzz`.
 set(LUA_C_API_TESTS_PREFIX ${PROJECT_BINARY_DIR}/test/fuzz/${LUA_C_API_TESTS_TARGET})
 
+# The test `luaL_loadbuffer_proto_test` is not enabled, because
+# it is quite similar to `luaL_loadbuffer_fuzzer` located in
+# test/fuzz/luaL_loadbuffer/.
 string(JOIN ";" LUA_C_API_TESTS
   ffi_cdef_proto_test
   lua_dump_test
   luaL_gsub_test
-  luaL_loadbuffer_proto_test
   luaL_loadbufferx_test
   lua_load_test
   luaL_traceback_test
```